_pages/team.html

```diff
@@ -140,10 +146,117 @@ <h5>Ricardo Quiroz </h5>
+            <h5>Tina Heartney</h5>
+            <p>Tina Heartney is a parent volunteer.</p>
+        </div>
+        <div>
+            <img src="/images/team/placeholder.png">
+            <p class="label">Parent</p>
+            <h5>Marvin Hom</h5>
+            <p>Marvin Hom is a parent volunteer.</p>
+        </div>
+        <div>
+            <img src="/images/team/placeholder.png">
+            <p class="label">Parent</p>
+            <h5>Sandy Lee</h5>
+            <p>Sandy Lee is a parent volunteer.</p>
+        </div>
+        <div>
+            <img src="/images/team/placeholder.png">
+            <p class="label">Parent</p>
+            <h5>Kate McTigue</h5>
+            <p>Kate McTigue is a parent volunteer.</p>
+        </div>
+        <div>
+            <img src="/images/team/placeholder.png">
+            <p class="label">Parent</p>
+            <h5>Abida Mohammad</h5>
+            <p>Abida Mohammad is a parent volunteer.</p>
+        </div>
+        <div>
+            <img src="/images/team/placeholder.png">
+            <p class="label">Parent</p>
+            <h5>Fakruddin Shaik</h5>
+            <p>Fakruddin Shaik is a parent volunteer.</p>
+        </div>
+        <div>
+            <img src="/images/team/placeholder.png">
+            <p class="label">Parent</p>
+            <h5>Michele Synnott</h5>
+            <p>Michele Synott is a parent volunteer.</p>
+        </div>
+        <div>
+            <img src="/images/team/placeholder.png">
+            <p class="label">Parent</p>
+            <h5>Stephen Synnott</h5>
+            <p>Stephen Synnott is a parent volunteer.</p>
+        </div>
+        <div>
+            <img src="/images/team/placeholder.png">
+            <p class="label">Parent</p>
+            <h5>Suzanne Updike</h5>
+            <p>Suzanne Updike is a parent volunteer.</p>
+        </div>
+        <div>
+            <img src="/images/team/placeholder.png">
+            <p class="label">Parent</p>
+            <h5>Tom Updike</h5>
+            <p>Tom Updike is a parent volunteer.</p>
+        </div>
+        <div>
+            <img src="/images/team/placeholder.png">
+            <p class="label">Parent</p>
+            <h5>Daniel Danckaert</h5>
+            <p>Daniel Danckaert is a parent volunteer.</p>
+        </div>
+        <div>
+            <img src="/images/team/placeholder.png">
+            <p class="label">Parent</p>
+            <h5>Jean Danckaert</h5>
+            <p>Jean Danckaert is a parent volunteer.</p>
+        </div>
+        <div>
+            <img src="/images/team/placeholder.png">
+            <p class="label">Parent</p>
+            <h5>Dale Eppler</h5>
+            <p>Dale Eppler is a parent volunteer.</p>
+        </div>
+        <div>
+            <img src="/images/team/placeholder.png">
+            <p class="label">Parent</p>
+            <h5>Sviatlana Eppler</h5>
+            <p>Sviatlana Eppler is a parent volunteer.</p>
+        </div>  
```

Comment:
Are we sure about including parents in the team page? Personally, I would advocate for adding a "mentors" page (even with similar formatting if you like) and listing them separately, rather than integrating them into this list. Up to you but I disagree with this choice.

Reply:
Ok I was just told to add the parents to it. I can make a separate tab